src/test/java/seedu/address/storage/JsonAdaptedGoodTest.java

```diff
@@ -29,22 +31,30 @@ public void toModelType_validGoodDetails_returnsGood() throws Exception {
     @Test
     public void toModelType_invalidGoodName_throwsIllegalValueException() {
         JsonAdaptedGood good =
-                new JsonAdaptedGood(INVALID_GOOD_NAME, VALID_GOOD_QUANTITY);
+                new JsonAdaptedGood(INVALID_GOOD_NAME, VALID_GOOD_QUANTITY, VALID_GOOD_THRESHOLD);
         String expectedMessage = GoodName.MESSAGE_CONSTRAINTS;
         assertThrows(IllegalValueException.class, expectedMessage, good::toModelType);
     }
 
     @Test
     public void toModelType_nullGoodName_throwsIllegalValueException() {
-        JsonAdaptedGood good = new JsonAdaptedGood(null, VALID_GOOD_QUANTITY);
+        JsonAdaptedGood good = new JsonAdaptedGood(null, VALID_GOOD_QUANTITY, VALID_GOOD_THRESHOLD);
         String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, GoodName.class.getSimpleName());
         assertThrows(IllegalValueException.class, expectedMessage, good::toModelType);
     }
 
```

Comment:
How about null threshold test?

Reply:
The required parameter is int, null cannot be thrown in as parameter. I tried, IDE gives me an error trying to set null for int parameter.